accounting/accounting_cli.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import sqlite3
+import argparse
+import time
+import sys
+
+import pandas as pd
+
+from accounting import accounting_cli_functions as aclif
+
+def msg(name=None):
+    return """
+    Description: accounting_cla will translate command line arguments into
+    SQLite instructions for the Flux Accounting Database.
+
+    Usage: python cla_hostlist.py [OPTION]...
+
+    -h, --help                              Display this message.
+    -u, --user USER                         View account information for a specific user.
+    -a, --add_user FIELDS                   Add a user to the accounting database.
+
+        FIELDS=user_name admin_level account parent_acct shares max_jobs max_wall_pj
+
+    -d, --delete_user USER                  Remove a user from the accounting database.
+    -e, --edit_value USER FIELD NEW_VALUE   Edit an existing user's value in the accounting database
+    """
```

Comment:
I think all of this should already be auto-generated by argparse.  Is there a particular part of the usage message that wasn't being properly generated?  If so, maybe we can fix that issue in a more generic way.

Reply:
Yup, you're absolutely right. I wasn't paying close enough attention and had manually written my own help message. With the help of `metavars` I think I was able to produce a much cleaner help message with `flux-accounting -h`:

```
usage: flux-accounting [-h] [-u USER_NAME]
                       [-a USER_NAME ADMIN_LEVEL ACCOUNT PARENT_ACCT SHARES MAX_JOBS MAX_WALL_PJ]
                       [-d USER_NAME] [-e USER_NAME FIELD NEW VALUE]

Description: Translate command line arguments into SQLite instructions for the
Flux Accounting Database.

optional arguments:
  -h, --help            show this help message and exit
  -u USER_NAME, --user USER_NAME
                        view a user's account information
  -a USER_NAME ADMIN_LEVEL ACCOUNT PARENT_ACCT SHARES MAX_JOBS MAX_WALL_PJ, --add-user USER_NAME ADMIN_LEVEL ACCOUNT PARENT_ACCT SHARES MAX_JOBS MAX_WALL_PJ
                        add a user to the accounting database
  -d USER_NAME, --delete-user USER_NAME
                        remove a user from the accounting database
  -e USER_NAME FIELD NEW VALUE, --edit-user USER_NAME FIELD NEW VALUE
                        edit a user's value
```